tests/test_series.py

```diff
@@ -1604,12 +1605,14 @@ def test_pandera_generic() -> None:
     T = TypeVar("T")
 
     class MySeries(pd.Series, Generic[T]):
-        ...
+        def __new__(cls, *args, **kwargs) -> Self:
+            return object.__new__(cls)
 
     def func() -> MySeries[float]:
         return MySeries[float]([1, 2, 3])
 
-    func()
+    result = func()
+    assert result.iloc[1] == 2
```

Comment:
I modified the test to make sure that defining `__new__()` didn't change the result.